sdk/README.md

```diff
@@ -1,25 +1,29 @@
 # Lookout Analyzer SDK
 
-An analyzer is a gRPC server, that implements [Analyzer service](https://github.com/src-d/lookout/tree/master/sdk/service_analyzer.proto) to receive events from the server.
+An analyzer is a [gRPC](https://grpc.io/) server that implements the [protocol buffers Analyzer service](./service_analyzer.proto) to receive events from the lookout server.
```

Comment:
I think that after gRPC implies protocol buffers so we can remove "protocol buffers"